docs/data/date-pickers/introduction/introduction.md

```diff
@@ -0,0 +1,144 @@
+---
+product: date-pickers
+title: Date and Time Picker - Introduction
+packageName: '@mui/x-date-pickers'
+githubLabel: 'component: pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.html
+---
+
+# Date and Time Pickers - Introduction
+
+<p class="description">The Date and Time pickers expose a lot of components to fit your every needs.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic usage
+
+All the components can be uncontrolled or controlled by using the `value` / `onChange` props:
+
+{{"demo": "ControlledComponent.js"}}
+
+## Imports
+
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but without the nested imports.
+
+For example, to use the `DatePicker` component, the following three imports are valid:
+
+```tsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { DatePicker } from '@mui/x-date-pickers';
+import { DatePicker } from '@mui/x-date-pickers-pro';
+```
+
+## Other components
+
+### Choose interaction style
+
+The first thing you need to decide, is how the user should be able to select their value:
+
+- For input editing with a popover, use the _Picker_ components:
+
+{{"demo": "BasicDatePicker.js", "hideToolbar": true, "bg": "inline"}}
+
+- For input-only editing, use the _Field_ components:
+
+{{"demo": "BasicDateField.js", "hideToolbar": true, "bg": "inline"}}
+
+- For inline editing, use the _Calendar / Clock_ components:
+
+{{"demo": "BasicDateCalendar.js", "hideToolbar": true, "bg": "inline"}}
+
+:::info
+Each _Picker_ is a combination of one _Field_ and one or several _Calendar / Clock_ components.
+For example, the `DatePicker` is the combination of the `DateField` and the `DateCalendar`.
+
+The _Calendar / Clock_ components are rendered inside a _Popover_ on desktop and inside a _Modal_ on mobile.
+:::
+
+### Date or time editing ?
+
+The Date and Time Pickers are divided into six families of components.
+The demo below shows each one of them using their field component:
+
+{{"demo": "ComponentFamilies.js"}}
```

Comment:
Again for this demo I would have the `<span className="plan-pro"/>` before the name and with a tooltip.

```suggestion
{{"demo": "ComponentFamilies.js"}}
```